firmware/application/apps/ui_coasterp.hpp

```diff
@@ -52,6 +52,7 @@ class CoasterPagerView : public View {
     tx_modes tx_mode = IDLE;
 
     TxRadioState radio_state_{
+        911600000, /* frequency */
```

Comment:
Was this set somewhere before? I don't see it in the PR.

Reply:
Actually that wasn't the default frequency I meant to paste in there, so good catch.  Seems that a common pager freq is the 433.92 so I've updated.